frontend/src/static/js/main.js

```diff
@@ -84,8 +97,8 @@ window.app = new window.Vue({
       this.isTabActive = true;
       this.isTabAuthorship = true;
       this.isCollapsed = false;
-      if (document.getElementById("tabs-wrapper")) {
-        document.getElementById("tabs-wrapper").scrollTop = 0;
+      if (document.getElementById('tabs-wrapper')) {
+        document.getElementById('tabs-wrapper').scrollTop = 0;
```

Comment:
Is there a reason for this change?

Reply:
Actually let me check what is the linting rule. I might have mindlessly changed double quote to single quote

Reply:
The linting rule is to use single-quote, not double-quote. I made this change in a separate commit (not mixed with functional change). Let me know if it is better to have dedicated PR for this kind of fix (linting)

Reply:
Hmmm, let's just leave this change here, since you are already editing the file and portions of the code nearby that area anyway.